### DISCUSSION TITLE
Parallel Latent Growth Model

### DISCUSSION THREAD

**Discussion by maupind** [insightsengineering/JointSurvivalModels.jl#49] [General]
Hi all,

I am just curious if this package can support parallel latent growth models (example here: https://academic.oup.com/gerontologist/article/62/3/352/6204639?login=false).

Thank you!

**Comment by ya0** [comment]
Hello there, sorry for the late reply.

This package is agnostic about the longitudinal model. If you can formulate the longitudinal model as a julia function you can use this package to build a joint model around it and then use Turing.jl to sample the parameters of the joint model including the parameters of the longitudinal model. 

I had a quick look at the paper but could not determine what the challenges are. Maybe you can tell me more?

**Reply by maupind** [reply]
No worries at all, not a big delay. I think the issue comes from my lack of understanding to be honest as joint modelling is new to me (as well as Bayesian analysis in general!). I believe I have found an R package that uses BUGS syntax to get this work, but I believe I was misinterpreting what I was reading in the documentation of packages such as this and others; assuming that because phrases like latent growth are not specifically utilised that they were not constructed with this purpose when in reality it seems to be more if you have the knowledge to write out the model in the desired code. 

Thank you for response, it seems I misinterpreted things and that this package would be useful for me! I have had some variation in model outputs with R packages so I think I will give this a shot and see how it compares.

**Reply by ya0** [reply]
Take a look at the example in the documentation https://insightsengineering.github.io/JointSurvivalModels.jl/dev/FirstExample/ and the short description in https://github.com/insightsengineering/JointSurvivalModels.jl/blob/main/paper/paper.md which is based on a paper about a mixed longitudinal model. This might give you some perspective what is possible using this code.
